### ISSUE TITLE
Pre-compiled distribution

### ISSUE THREAD

**Issue by joaocc** [DIY-able/DynamicDnsUpdater.Service#1]
Hi,
I've been looking for this since I started using Route53.
Do you have anything against making a binary version available for download?
How can I help?
Thx

**Comment by DIY-able** [comment]
Yes, binary is now on Code Project page, with updated bug fixes and minor enhancement. 
http://www.codeproject.com/Articles/882487/Amazon-AWS-Route-Dynamic-IP-Updater-Windows-Servic
